src/DTNtuplePh2TPGThetaFiller.cc

```diff
@@ -0,0 +1,134 @@
+/** \class DTNtuplePh2TPGThetaFiller DTNtuplePh2TPGThetaFiller.cc DTDPGAnalysis/DTNtuples/src/DTNtuplePh2TPGThetaFiller.cc
+ *  
+ * Helper class : the Phase-2 DT Theta TP filler
+ *
+ * \author J. Leon (CIEMAT, Madrid)
+ *
+ *
+ */
+
+#include "DTDPGAnalysis/DTNtuples/src/DTNtuplePh2TPGThetaFiller.h"
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EDAnalyzer.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+
+#include <iostream>
```

Comment:
not sure, I would have to test it. It is there because it was copied from DTNtuplePh2TPGPhiFiller.cc. Do you have some instructions to perform a test to check those things?

Reply:
Well, if you remove it and the code complies you are done with testing it.
I would try to complie the ntuples alone (no phase-2 emulator or unpacker) in a recent _CMSSW_12_4_X_ .